fastrpc.cmake

```diff
@@ -74,11 +74,9 @@ else()
 endif()
 
 set(ADSPMSGD ${HEXAGON_SDK_ROOT}/${SDKLIB}/common/adspmsgd/ship/UbuntuARM_${RELEASE}/adspmsgd.a)
-set(RPCMEM ${HEXAGON_SDK_ROOT}/${SDKLIB}/common/rpcmem/UbuntuARM_${RELEASE}/rpcmem.a)
```

Comment:
How would anyone know where to find this library if you remove the variable? How would they know that they need it? What does it break for 8096?